schemas/http.yml

```
@@ -15,9 +15,16 @@
       description: >
         HTTP request method.
 
-        The field value must be normalized to lowercase for querying. See
-        the documentation section "Implementing ECS".
-      example: get, post, put
+        Prior to ECS 1.6.0 the following guidance was provided:
+
+          The field value must be normalized to lowercase for
+          querying. See the documentation section "Implementing ECS".
+
+        As of ECS 1.6.0, the guidance is deprecated because the
+        original case of the method may be useful in anomaly
+        detection.  Original case will be mandated in ECS 2.0.0
+
+      example: get, POST, Put
```

Nitpick: since we favour original casing, which will almost always be uppercase here I'd use all caps for each, and add one to make the point about anomaly detection.

```suggestion
      example: GET, POST, PUT, PoST
```